src/Utility/HttpResponsePaginator.php

```diff
@@ -298,10 +304,12 @@ private function processLastResponse(): bool
 
                 $start = (int) $start;
 
+                // @codeCoverageIgnoreStart
```

Comment:
Similar question as above, wondering why we don't test or monitor coverage for this

Reply:
Same as above 👍